public/client.js

```diff
@@ -1,9 +1,19 @@
-// $(function () {
-// 	$.get('/trucks', function (trucks) {
-// 		var truckList = [];
-// 		trucks.forEach(function (truck) {
-// 			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
-// 		});
-// 		$('.truck-list').append(truckList);
-// 	});
-// });
\ No newline at end of file
+$(function () {
+	
+	$.get('/trucks', function (trucks) {
+		var truckList = [];
+		trucks.forEach(function (truck) {
+			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
+		});
+		$('.truck-list').append(truckList);
+	});
+
+	$.get('/food-types', function (foodTypes) {
+		var foodTypeList = [];
+		foodTypes.forEach(function (type) {
+			foodTypeList.push('<li><a href="/food-types/' + type + '/">' + type + '</a></li>')
```

Comment:
This is a tricky part because some of the food types have a '/' in them, which makes routing to them problematic. This can be handled by encoding the type here: `encodeURIComponent(type)`, then decoding in the route handler in app.js.